src/metabase/models/timeline.clj

```diff
@@ -35,6 +37,65 @@
     (assoc timeline :collection (root-collection))
     timeline))
 
+
+;;;; hydration
+
+(defn- fetch-events
+  "Fetch events for timelines in `timeline-ids`. Can include optional `start` and `end` dates in the options map, as
+  well as `all?`. By default, will return only unarchived events, unless `all?` is truthy and will return all events
+  regardless of archive state."
+  [timeline-ids {:events/keys [all? start end]}]
+  (let [timeline-icon {:select [[(hsql/call :coalesce :icon DefaultIcon) :tl_icon]
+                                [:id :tl_id]]
+                       :from   [Timeline]}
+        query         {:select    [[(hsql/call :coalesce :icon :tl_icon) :icon]
+                                   :description :archived :timezone :time_matters :name
+                                   :timeline_id :creator_id :updated_at :id :timestamp :created_at]
```

Comment:
Here I explicitly select all of the keys for timeline events.
Before, I had `[[(hsql/call :coalesce :icon :tl_icon) :icon] :*]` which worked with my PostgreSQL app-db, but started failing in CI with the mariadb tests. It's apparently a syntax error with Mariadb.